packages/components/autocomplete/__tests__/autocomplete.test.tsx

```diff
@@ -357,12 +358,14 @@ describe('Autocomplete.vue', () => {
         </FormItem>
       ))
 
-      await nextTick()
+      vi.useRealTimers()
+      await sleep()
       const formItem = wrapper.find('[data-test-ref="item"]')
-      const input = await wrapper.find('[data-test-ref="input"]')
+      const input = wrapper.find('[data-test-ref="input"]')
       const formItemLabel = formItem.find('.el-form-item__label')
       expect(formItem.attributes().role).toBeFalsy()
       expect(formItemLabel.attributes().for).toBe(input.attributes().id)
+      vi.useFakeTimers()
```

Comment:
I do not quite understand why this modification is necessary and how much it has to do with the PR title

Reply:
Initially `isShouldAssociated` value is `false`,  so `ormItem.attributes().role` has the value of `group`,  After the execution of `associatedElement` method, `isShouldAssociated` is true, then the test can pass.